src/tablebody/tablebody.test.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import UserList from'./tablebody'
+import { shallow, render, mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import enzyme from 'enzyme';
+import { TableBody } from "@material-ui/core";
+
+enzyme.configure({ adapter: new Adapter() })
```

Comment:
we have already configured adapter in test-setup

Reply:
yeah sir, I will remove this adpator